readme.md

```diff
@@ -0,0 +1,171 @@
+## Description
+
+This is an SDK for interacting with [CMS Blue Button 2.0 API](https://bluebutton.cms.gov/developers/),
+the API allows applications to obtain a beneficiary's (who has login account with medicare.gov) grant
+to access his/her medicare claims data - through OAUTH2 [(RFC 6749)](https://datatracker.ietf.org/doc/html/rfc6749) authorization flow.
+
+By using the SDK, the development of applications accessing Blue Button 2.0 API can be greatly simplified.
+
+Note, following the OAUTH2 best practices, OAUTH2 PKCE etension [(RFC 7636)](https://datatracker.ietf.org/doc/html/rfc7636) is always enabled.
+
+## Installation
+
+Using npm:
```

Comment:
We can also add something about using Typescript, where they'll want to use `npm install --save-dev @types/cms-bluebutton-sdk` or `yarn add --dev @types/cms-bluebutton-sdk` for yarn.